<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_Symulate">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Symulate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_SetRange">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Range">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Set time of symulation (max 100)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Range">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Name">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_SetName">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Name">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Set a name of the city</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Insurance">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Set an insurance of cars (max 100)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Insurance">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_SetInsurance">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_guitarGif">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>39</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_carGif">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>39</width>
      <height>11</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionWczytaj"/>
   </widget>
   <widget class="QMenu" name="menuOkno">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionZamknij"/>
    <addaction name="actionZminimalizuj"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuOkno"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionWczytaj">
   <property name="text">
    <string>Load</string>
   </property>
   <property name="toolTip">
    <string>Load saved city</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionZamknij">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionZminimalizuj">
   <property name="text">
    <string>Minimalize</string>
   </property>
   <property name="toolTip">
    <string>Minimalize window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
